go.sum

```diff
@@ -1,945 +1,21 @@
-cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Any idea why a small change to the health state enum caused so many changes here?

Reply:
I updated my branch and go sum is now updated and almost all the changes in go sum are reverted